{
    "data": [
        {
            "test_name": "AlgorithmsTest_SassenaFFTTest",
            "latest_failed_build": "611",
            "agent_name": "Mac",
            "last_failed_outcome": "Flake",
            "pipeline_name": "main_nightly_deployment_prototype",
            "problem_count": 1,
            "past_outcome": {
                "build_keys": [
                    "611",
                    "610",
                    "609",
                    "608",
                    "607",
                    "606",
                    "605",
                    "604",
                    "603",
                    "602",
                    "598"
                ],
                "past_outcome_list": [
                    "Flake",
                    "None",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed"
                ]
            },
            "last_stack_traces": {
                "test_name": "AlgorithmsTest_SassenaFFTTest",
                "build": "611",
                "agent_name": "Mac",
                "trial": 1,
                "outcome": "Failed",
                "test_time": "0.48",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.7.20230808.1601.dev33\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\nDllOpen-[Error] Could not open library /Users/builder/Jenkins/workspace/main_nightly_deployment_prototype/source/build/bin/libMantidPythonInterfaceCore.dylib: dlopen(/Users/builder/Jenkins/workspace/main_nightly_deployment_prototype/source/build/bin/libMantidPythonInterfaceCore.dylib, 10): Symbol not found: _PyBaseObject_Type\n  Referenced from: /Users/builder/Jenkins/workspace/main_nightly_deployment_prototype/source/mambaforge/envs/mantid-developer/lib/libboost_python38.dylib\n  Expected in: flat namespace\n in /Users/builder/Jenkins/workspace/main_nightly_deployment_prototype/source/mambaforge/envs/mantid-developer/lib/libboost_python38.dylib\nRunning 3 tests.SassenaFFT-[Notice] SassenaFFT started\nAlgorithmsTest(3988,0x70000e182000) malloc: *** error for object 0x7fb171439288: incorrect checksum for freed object - object was probably modified after being freed.\n*** set a breakpoint in malloc_error_break to debug\nSubprocess aborted\n\n          Start  419: AlgorithmsTest_SassenaFFTTest\n"
            }
        }
    ]
}